modules/surface_matching/samples/ppf_load_match.cpp

```diff
@@ -106,7 +106,7 @@ int main(int argc, char** argv)
     cout << endl << "Starting matching..." << endl;
     vector<Pose3DPtr> results;
     tick1 = cv::getTickCount();
-    detector.match(pcTest, results, 1.0/40.0, 0.05);
+    detector.match(pcTest, results, 1.0/4.0, 0.05);
```

Comment:
Well, for this data both values should work. Some tougher scenarios would require higher values.